pandas-stubs/io/formats/style.pyi

```diff
@@ -1,97 +1,267 @@
         index: bool = ...,
-        index_label=...,
+        index_label: IndexLabel | None = ...,
         startrow: int = ...,
         startcol: int = ...,
-        engine=...,
+        engine: str | None = ...,
         merge_cells: bool = ...,
-        encoding=...,
+        encoding: str | None = ...,
         inf_rep: str = ...,
         verbose: bool = ...,
-        freeze_panes=...,
+        freeze_panes: tuple[int, int] | None = ...,
     ) -> None: ...
-    def format(self, formatter, subset=..., na_rep: str | None = ...): ...
-    def render(self, **kwargs): ...
-    def __copy__(self): ...
-    def __deepcopy__(self, memo): ...
+    def to_latex(
+        self,
+        buf: FilePath | WriteBuffer[str] | None = ...,
+        *,
+        column_format: str | None = ...,
+        position: str | None = ...,
+        position_float: str | None = ...,
+        hrules: bool | None = ...,
+        clines: str | None = ...,
+        label: str | None = ...,
+        caption: str | tuple | None = ...,
+        sparse_index: bool | None = ...,
+        sparse_columns: bool | None = ...,
+        multirow_align: str | None = ...,
+        multicol_align: str | None = ...,
+        siunitx: bool = ...,
+        environment: str | None = ...,
+        encoding: str | None = ...,
+        convert_css: bool = ...,
+    ): ...
+    def to_html(
+        self,
+        buf: FilePath | WriteBuffer[str] | None = ...,
+        *,
+        table_uuid: str | None = ...,
+        table_attributes: str | None = ...,
+        sparse_index: bool | None = ...,
+        sparse_columns: bool | None = ...,
+        bold_headers: bool = ...,
+        caption: str | None = ...,
+        max_rows: int | None = ...,
+        max_columns: int | None = ...,
+        encoding: str | None = ...,
+        doctype_html: bool = ...,
+        exclude_styles: bool = ...,
+        **kwargs,
+    ): ...
+    def set_td_classes(self, classes: DataFrame) -> Styler: ...
+    def __copy__(self) -> Styler: ...
+    def __deepcopy__(self, memo) -> Styler: ...
     def clear(self) -> None: ...
-    def apply(self, func, axis: int = ..., subset=..., **kwargs): ...
-    def applymap(self, func, subset=..., **kwargs): ...
-    def where(self, cond, value, other=..., subset=..., **kwargs): ...
-    def set_precision(self, precision): ...
-    def set_table_attributes(self, attributes): ...
-    def export(self): ...
-    def use(self, styles): ...
-    def set_uuid(self, uuid): ...
-    def set_caption(self, caption): ...
-    def set_table_styles(self, table_styles): ...
-    def set_na_rep(self, na_rep: str) -> Styler: ...
-    def hide_index(self): ...
-    def hide_columns(self, subset): ...
-    def highlight_null(self, null_color: str = ...): ...
+    def apply(
+        self,
+        func: Callable,
+        axis: Axis | None = ...,
+        subset: Subset | None = ...,
+        **kwargs,
+    ) -> Styler: ...
+    def apply_index(
+        self,
+        func: Callable,
```

Comment:
`Callable[[Series], list[str]]` ??

Reply:
The types in the PR are the actual ones supported.